07.ls_object/lib/file_entry.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+require 'etc'
+require 'time'
+class FileEntry < File::Stat
+  attr_reader :name
+
+  def initialize(file_path)
+    super
+    @name = File.basename file_path
+  end
+
+  def permission
+    mode_to_s(mode.to_s(8).match(/.{4}$/)[0].chars)
+  end
+
+  def owner
+    Etc.getpwuid(uid).name
+  end
+
+  def group
+    Etc.getgrgid(gid).name
+  end
+
+  def update_time
+    Time.parse(atime.to_s).strftime('%_m %_d %H:%M')
```

Comment:
```suggestion
    atime.strftime('%_m %_d %H:%M')
```

Time→文字列→Timeと変換する必要はないような？

あと、以下の記事も読んでみてほしいです。
https://bootcamp.fjord.jp/pages/ideal-data-type